resources/schema.json

```diff
@@ -614,7 +633,10 @@
           "items": {
             "$id": "#/appliesTo/items",
             "type": "string",
-            "pattern": "^(bpmn:)"
+            "pattern": "^(bpmn:)",
+            "errorMessage": {
+              "pattern": "invalid item for \"appliesTo\", should match pattern ^(bpmn:)"
```

Comment:
Can technically also match things like `camunda:ServiceTaskLike`. It should be a namespaced property name, at least.